qml/components/GamePreview.qml

```diff
@@ -0,0 +1,126 @@
+/*
+    Copyright (C) 2020 Sebastian J. Wolf and other contributors
+
+    This file is part of Fernschreiber.
+
+    Fernschreiber is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Fernschreiber is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Fernschreiber. If not, see <http://www.gnu.org/licenses/>.
+*/
+import QtQuick 2.6
+import Sailfish.Silica 1.0
+import WerkWolf.Fernschreiber 1.0
+import "../js/functions.js" as Functions
+import "../js/twemoji.js" as Emoji
+
+Column {
+    id: gamePreviewItem
+
+    property ListItem messageListItem
+    property MessageOverlayFlickable overlayFlickable
+    property var rawMessage: messageListItem ? messageListItem.myMessage : overlayFlickable.overlayMessage
+    property bool highlighted
+
+    width: parent.width
+    height: childrenRect.height
+
+
+    Label {
+        width: parent.width
+        font.bold: true
+        font.pixelSize: Theme.fontSizeSmall
+        text: Emoji.emojify(rawMessage.content.game.title || "", font.pixelSize)
+        truncationMode: TruncationMode.Fade
+        textFormat: Text.StyledText
+        wrapMode: Text.Wrap
+    }
+    Label {
+        width: parent.width
+        font.pixelSize: Theme.fontSizeExtraSmall
+        text: Emoji.emojify(rawMessage.content.game.description || "", font.pixelSize)
+        truncationMode: TruncationMode.Fade
+        textFormat: Text.StyledText
+        wrapMode: Text.Wrap
+    }
+    Label {
+        width: parent.width
+        font.pixelSize: Theme.fontSizeExtraSmall
+        text: Emoji.emojify(Functions.enhanceMessageText(rawMessage.content.game.text) || "", font.pixelSize)
+        truncationMode: TruncationMode.Fade
+        wrapMode: Text.Wrap
+        textFormat: Text.StyledText
+        onLinkActivated: {
+            var chatCommand = Functions.handleLink(link);
+            if(chatCommand) {
+                tdLibWrapper.sendTextMessage(chatInformation.id, chatCommand);
+            }
+        }
+    }
+    Item {
+        width: parent.width
+        height: Theme.paddingLarge
+    }
+
+    Image {
+        id: thumbnail
+        source: thumbnailFile.isDownloadingCompleted ? thumbnailFile.path : ""
+        fillMode: Image.PreserveAspectCrop
+        asynchronous: true
+        width: visible ? parent.width : 0
```

Comment:
This is strange. First, invisible items should be completely ignored by the Column, their size doesn't matter. Second, what sets `visible` for this item? Did you forget `visible: opacity > 0`?

Reply:
You are absolutely right on both accounts. I mixed this up when changing my initial anchor based layout to a less sophisticated/easier column. 
Thank you!